colour/io/luts/tests/test_clf.py

```diff
@@ -0,0 +1,548 @@
+        Example 5.
+        """
+        example = """
+        <Range inBitDepth="10i" outBitDepth="10i">
+            <Description>10-bit full range to SMPTE range</Description>
+            <minInValue>0</minInValue>
+            <maxInValue>1023</maxInValue>
+            <minOutValue>64</minOutValue>
+            <maxOutValue>940</maxOutValue>
+        </Range>
+        """
+        doc = parse_clf(EXAMPLE_WRAPPER.format(example))
+        node = doc.process_nodes[0]
+        self.assertIsInstance(node, clf.Range)
+        self.assertEqual(node.id, None)
+        self.assertEqual(node.name, None)
+        self.assertEqual(node.in_bit_depth, clf.BitDepth.i10)
+        self.assertEqual(node.out_bit_depth, clf.BitDepth.i10)
+        self.assertEqual(node.description, "10-bit full range to SMPTE range")
+        (self.assertEqual(node.min_in_value, 0.0),)
+        (self.assertEqual(node.min_out_value, 64.0),)
+        (self.assertEqual(node.max_out_value, 940.0),)
+
+    def test_log_example_1(self):
+        """
+        Test parsing of the example process node from the official CLF specification
+        Example 6.
+        """
+        example = """
+        <Log inBitDepth="16f" outBitDepth="16f" style="log10">
+            <Description>Base 10 Logarithm</Description>
+        </Log>
+        """
+        doc = parse_clf(EXAMPLE_WRAPPER.format(example))
+        node = doc.process_nodes[0]
+        self.assertIsInstance(node, clf.Log)
+        self.assertEqual(node.id, None)
+        self.assertEqual(node.name, None)
+        self.assertEqual(node.in_bit_depth, clf.BitDepth.f16)
+        self.assertEqual(node.out_bit_depth, clf.BitDepth.f16)
+        self.assertEqual(node.description, "Base 10 Logarithm")
+        (self.assertEqual(node.style, clf.LogStyle.LOG_10),)
+        (self.assertEqual(node.log_params, None),)
+
+    def test_log_example_2(self):
+        """
+        Test parsing of the example process node from the official CLF specification
+        Example 7.
+        """
+        example = """
+        <Log inBitDepth="32f" outBitDepth="32f" style="cameraLinToLog">
+            <Description>Linear to DJI D-Log</Description>
+            <LogParams base="10" logSideSlope="0.256663" logSideOffset="0.584555"
+                linSideSlope="0.9892" linSideOffset="0.0108" linSideBreak="0.0078"
+                linearSlope="6.025"/>
+        </Log>
+        """
+        doc = parse_clf(EXAMPLE_WRAPPER.format(example))
+        node = doc.process_nodes[0]
+        self.assertIsInstance(node, clf.Log)
+        self.assertEqual(node.id, None)
+        self.assertEqual(node.name, None)
+        self.assertEqual(node.in_bit_depth, clf.BitDepth.f32)
+        self.assertEqual(node.out_bit_depth, clf.BitDepth.f32)
+        self.assertEqual(node.description, "Linear to DJI D-Log")
+        (self.assertEqual(node.style, clf.LogStyle.CAMERA_LIN_TO_LOG),)
+        self.assertAlmostEqual(node.log_params.base, 10.0)
```

Comment:
This assertion would fail if the `"urn:NATAS:AMPAS:LUT:v2.0"` bit were removed from EXAMPLE_WRAPPER